spec/request/api/v1/confirmations/show_confirmation_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+describe 'GET /api/v1/auth/confirmation', type: :request do
+    let(:user) { create(:user, { password: 'password' } ) }
+
+    it 'returns the user is not confirmed' do
+      expect(user.confirmed?).to be false
+    end
+
+    it 'returns that a confirmation email was sent' do
+      post user_session_path, params: {email: user.email, password: "password"}, as: :json
```

Comment:
I would move this to another request test as it is part of the sign in

Reply:
In the describe of the test you said you are testing a `GET` not a `POST`

Reply:
Also, use single quotes for simple strings